lib/card_validation/validity.rb

```diff
@@ -51,6 +51,14 @@ def double
     digits.reverse.each_with_index.map { |x, i| i.even? ? x * 2 : x }
   end
 
+  def empty?
+    number.respond_to?(:empty?) && number.empty?
```

Comment:
Good catch! https://github.com/onthebeach/card_validation/pull/2/commits/6738648b60af6fbc7426d58d5285ef73933ebf49 fixes this.